docs/source/model_doc/swin.mdx

```diff
@@ -34,6 +34,8 @@ The hierarchical design and the shifted window approach also prove beneficial fo
 
 Tips:
 - One can use the [`AutoFeatureExtractor`] API to prepare images for the model.
+- Swin pads the inputs supporting any input size (if divisible by `32`).
+- Swin can be used as a *backbone*, when when `output_hidden_states = True` it will outputs both `hidden_states` and `reshaped_hidden_states`. `reshaped_hidden_states` have a size of `batch, channels, height, width`.
```

Comment:
```suggestion
- Swin can be used as a *backbone*, when `output_hidden_states = True` it will output both `hidden_states` and `reshaped_hidden_states`. The `reshaped_hidden_states` have a shape of `(batch, channels, height, width)`.
```

Reply:
my god, thanks! 🙏